Source/WebCore/platform/network/NetworkStorageSession.cpp

```diff
@@ -366,20 +360,6 @@ void NetworkStorageSession::resetAppBoundDomains()
 }
 #endif
 
-std::optional<Seconds> NetworkStorageSession::clientSideCookieCap(const RegistrableDomain& firstParty, std::optional<PageIdentifier> pageID) const
-{
-    if (!m_ageCapForClientSideCookies || !pageID || m_navigatedToWithLinkDecorationByPrevalentResource.isEmpty())
-        return m_ageCapForClientSideCookies;
-
-    auto domainIterator = m_navigatedToWithLinkDecorationByPrevalentResource.find(*pageID);
-    if (domainIterator == m_navigatedToWithLinkDecorationByPrevalentResource.end())
-        return m_ageCapForClientSideCookies;
-
-    if (domainIterator->value == firstParty)
-        return m_ageCapForClientSideCookiesShort;
-
-    return m_ageCapForClientSideCookies;
-}
```

Comment:
I wonder if we should leave this code for non-Cocoa ports. They don't have the cookie jar functionality to delete only client-side cookies.

Reply:
good point, will fix.

Reply:
If it's only ever called from NetworkStorageSessionCocoa then maybe we could move it to NetworkStorageSessionCocoa.  Let's not worry about that right now though.